src/main/java/editCard/EditCardInteractor.java

```diff
@@ -1,19 +1,20 @@
-package editCard;/*
+/*
  * Author: Albert Yan
  * Date: October 31/22
  * The use case interactor for editing cards given an input boundary
  */
 
```

Comment:
see comment in the EditCardController class

Reply:
Done.